OptimizelySDK.Tests/ProjectConfigTest.cs

```diff
@@ -810,5 +812,55 @@ public void TestVariationFeatureEnabledProperty()
             var variation = Config.GetVariationFromKey("test_experiment", "control");
             Assert.IsFalse(variation.IsFeatureEnabled);
         }
+
+        [Test]
+        public void TestBotFilteringValues()
+        {
+            // Verify that bot filtering value is true as defined in Config data.
+            Assert.True(Config.BotFiltering.GetValueOrDefault());
+
+            // Remove botFilering node and verify returned value in null.
+            JObject projConfig = JObject.Parse(TestData.Datafile);
+            if (projConfig.TryGetValue("botFiltering", out JToken token))
+            {
+                projConfig.Property("botFiltering").Remove();
+                var configWithoutBotFilter = ProjectConfig.Create(JsonConvert.SerializeObject(projConfig),
+                    LoggerMock.Object, ErrorHandlerMock.Object);
+
+                // Verify that bot filtering is null when not defined in datafile.
+                Assert.Null(configWithoutBotFilter.BotFiltering);
+            }
+        }
+
+        [Test]
+        public void TestGetAttributeIdWithReservedPrefix()
+        {
+            // Verify that attribute key is returned for reserved attribute key.
+            Assert.AreEqual(Config.GetAttributeId(EventBuilder.USER_AGENT_ATTRIBUTE), EventBuilder.USER_AGENT_ATTRIBUTE);
+
+            // Verify that attribute Id is returned for attribute key with reserved prefix that does not exist in datafile.
+            Assert.AreEqual(Config.GetAttributeId(EventBuilder.USER_AGENT_ATTRIBUTE), EventBuilder.USER_AGENT_ATTRIBUTE);
```

Comment:
nit. This test is no different from the previous test. Let's have a separate example. Like lets test for $opt_reserved_prefix_attribute